src/htm/engine/Region.cpp

```diff
@@ -514,6 +514,10 @@ void Region::setParameterArray(const std::string &name, const Array &array) {
   impl_->setParameterArray(name, (Int64)-1, array);
 }
 
+size_t Region::getParameterArrayCount(const std::string &name) {
+  return impl_->getParameterArrayCount(name, (Int64)-1);
```

Comment:
- nit, the one-liner method could probably have it's implementation in the hpp

Reply:
>  different set of py::format_descriptor formats

It turns out that numpy and python don't handle the type codes quite the same way for Int64. So when I get the code to see what C++ type it is I get different results on Linux vs Windows (l vs q).  So I have to check the size of the element as well.  

see https://docs.python.org/3.7/library/array.html